app/Category.php

```diff
@@ -14,4 +14,14 @@ public function places()
     {
         return $this->belongsToMany(Place::class);
     }
+
+
+    /**
+     * Method returning the page title of the object
+     * @return string
+     */
+    public function getPageTitle(): string
+    {
+        return $this->name . ' - Catégorie - ' . config('app.name', '');
```

Comment:
Opening discussion here, but do you think it makes the title a bit too long?

Reply:
Yes. I would remove `- Catégorie -` from the title to just get the game and app name.

Reply:
Done